boards/nuvoton/numaker_m2l31ki/doc/index.rst

```diff
@@ -0,0 +1,95 @@
+﻿.. _nuvoton_m2l31ki:
+
+NUVOTON NUMAKER M2L31KI
+########################
+
+Overview
+********
+
+The NuMaker M2L31KI is an Internet of Things (IoT) application focused platform
+specially developed by Nuvoton. The NuMaker-M2L31KI is based on the NuMicro® M2L31
+series MCU with ARM® -Cortex®-M23 core.
+
+.. image:: ./m2l31ki.jpg
```

Comment:
`/home/runner/_work/zephyr/zephyr/doc/_build/src/boards/nuvoton/numaker_m2l31ki/doc/index.rst:: WARNING: image file not readable: ../../../boards/nuvoton/numaker_m2l31ki/doc/m2l31ki.jpg`

webp?

Reply:
Yes, it's webp. Sorry forget to modify index.rst